packages/react-scripts/scripts/start.js

```diff
@@ -31,6 +32,8 @@ var prompt = require('react-dev-utils/prompt');
 var config = require('../config/webpack.config.dev');
 var paths = require('../config/paths');
 
+var isInteractive = tty.isatty(1);
```

Comment:
[`tty.isatty(fd)` should be passed a file descriptor](https://nodejs.org/api/tty.html#tty_tty_isatty_fd).